docs/kgw-cli.md

```diff
@@ -0,0 +1,9 @@
+# KGW CLI
+The KGW (Kusk Gateway) CLI is a helper utility used to generate instances of API Custom Resources
+from a given API spec.
+
+Using this utility removes the need to manually embed your OpenAPI document inside the Custom Resource and not worry about whether or not you have indented your spec enough.
+
+Another advantage of the kgw cli is how it can be used in your CI/CD pipelines. As you iterate on your OpenAPI spec and push those changes to Git, kgw can be used to automatically generate new API resources for Kusk Gateway.
+
+For more details, check out the [Github Repo](https://github.com/kubeshop/kgw)
```

Comment:
Let's also add a link to the petstore-cicd repo - i.e. 
"For a working example of how to use kgw with GitHub Actions to automate the deployment of an OpenAPI definition to Kusk Gateway, have a look at ... "